src/sdk/object_sdk.js

```diff
@@ -160,6 +160,11 @@ class ObjectSDK {
         return bucket.namespace;
     }
 
+    async read_force_md5_etag(name) {
+        const { bucket } = await bucket_namespace_cache.get_with_cache({ sdk: this, name });
+        return bucket.force_md5_etag;
+    }
+
```

Comment:
I don't think it makes sense that we'll add a new function for every new bucket option...
Can we instead add it to the namespace object when we initialize it?

Reply:
@achouhan09 you can propagate it to NamespaceFS [here](https://github.com/achouhan09/noobaa-core/blob/030b5603ac43611823d0c20cfd8261af8604caf9/src/sdk/object_sdk.js#L436)

Reply:
Done, Thanks